lametro/templates/search/search.html

```diff
@@ -1,6 +1,279 @@
 {% extends "search/search.html" %}
 
 {% load staticfiles %}
+{% load extras %}
+
+{% block content %}
+
+    <br/>
+    <div class='jumbotron' id='search-jumbotron'>
+        <!-- Add partial, so the title above search can be easily customized for site. -->
+        {% include 'partials/search_header.html' %}
+
+        <form method="get" action=".">
+            <div class="input-group site-intro-search">
+
+                {% for facet, selections in selected_facets.items %}
+                    {% for s in selections %}
+                        <input name="selected_facets" type="hidden" class="form-control" value="{{facet}}_exact:{{s}}">
+                    {% endfor %}
+                {% endfor %}
+
+                {% if 'sort_by=date' in request.get_full_path %}
+                    <input name="sort_by" type="hidden" class="form-control" value="date">
+                {% endif %}
+
+                {% if 'sort_by=title' in request.get_full_path %}
+                    <input name="sort_by" type="hidden" class="form-control" value="title">
+                {% endif %}
+
+                {% if 'sort_by=relevance' in request.get_full_path %}
+                    <input name="sort_by" type="hidden" class="form-control" value="relevance">
+                {% endif %}
+
+                {% if 'order_by=asc' in request.get_full_path %}
+                    <input name="order_by" type="hidden" class="form-control" value="asc">
+                {% endif %}
+
+                {% if 'order_by=desc' in request.get_full_path %}
+                    <input name="order_by" type="hidden" class="form-control" value="desc">
+                {% endif %}
+
+                {% with formatted_q=request.GET.q|remove_question %}
+                    {% include 'partials/search_bar.html' %}
+                {% endwith %}
+            </div>
+        </form>
+    </div>
+
+    <div class="row" style="margin-left:unset;">
```

Comment:
the left side of the search results is not even with the left side of the search bar element. this fixes that.